### PR TITLE
EL-4057 - Dashboard Layout not getting updated when using OnPush

### PR DESCRIPTION
**Pull request by DeanHaire**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4057

#### Description of Proposed Changes

- `renderDashboard()` when a new layout value is emitted
- Add testpage for layout which uses the async pipe

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4057-dashboard-layout

#### Downstream Build(s)

* [caf/ux-aspects-micro-focus](https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-4057-dashboard-layout~CI/)

### CODE REVIEW

**Review by donna-marijne — CHANGES_REQUESTED** [review]
Fix looks good, seems like some mistakenly removed tests here though?

File: e2e/tests/components/dashboard/dashboard.e2e-spec.ts [hunk 1]
```diff
@@ -372,114 +372,6 @@ describe('Dashboard Tests', () => {
         expect(await imageCompareFullPageScreen('dashboard-stacked-mode-widget-order')).toEqual(0);
     });
 
-    it('should not change the order of widgets when moving from regular to stacked mode', async () => {
```
[the remark this answers is not in the document]
Comment (DeanHaire) [reply e2e/tests/components/dashboard/dashboard.e2e-spec.ts:375]: These tests seemed to be duplicates of the test above this, unless there's some minor difference that I've missed. If you view the whole file you can see that I've left one copy of this test.

[the remark this answers is not in the document]
Comment (donna-marijne) [reply e2e/tests/components/dashboard/dashboard.e2e-spec.ts:375]: Ah, I see, no problem then!

**Review by donna-marijne — APPROVED** [review]
Looks good, thank you.

**Pull request merged**